eslint-plugin/docs/rules/avoid-autoplay.md

```diff
@@ -0,0 +1,47 @@
+# Disallow autoplay and enforce preload='none' for video and audio elements (`@ecocode/avoid-autoplay`)
+
+⚠️ This rule _warns_ in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
+
+⚠️ This rule _warns_ in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
```

Comment:
This generated header has been added twice.. can you remove one of it please?
(or all of them and re-run npm update script)